Src/FluentAssertions/Xml/XDocumentAssertions.cs

```diff
@@ -237,6 +238,71 @@ public AndConstraint<XDocumentAssertions> NotBeEquivalentTo(XDocument unexpected
             return new AndWhichConstraint<XDocumentAssertions, XElement>(this, xElement);
         }
 
+        /// <summary>
+        /// Asserts that the <see cref="XDocument.Root"/> element of the current <see cref="XDocument"/> has a single
+        /// child element with the specified <paramref name="expected"/> name.
+        /// </summary>
+        /// <param name="expected">
+        /// The full name <see cref="XName"/> of the expected child element of the current document's Root <see cref="XDocument.Root"/> element.
+        /// </param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <paramref name="because" />.
+        /// </param>
+        public AndConstraint<XDocumentAssertions> HaveSingleElement(XName expected, string because = "", params object[] becauseArgs)
```

Comment:
Should return a `AndWhichConstraint<XDocumentAssertions, XElement>` like `HaveElement`.

Reply:
And use the `Which` that `AndWhichConstraint` exposes in your test